Model/EasyshipCarrier.php

```diff
@@ -367,15 +369,20 @@ protected function _getQuotes()
          * @var \Magento\Shipping\Model\Rate\Result $result
          * @var \Magento\Quote\Model\Quote\Address\RateResult\Method $method
          */
+        $currentCurrencyCode = $this->_storeManager->getStore()->getCurrentCurrencyCode();
+        $baseCurrency = $this->_storeManager->getStore()->getBaseCurrency();
+        $convertRate = $baseCurrency->getRate($currentCurrencyCode);
+
         $result = $this->_rateResultFactory->create();
         foreach ($prefer_rates as $rate) {
+            $convertedPrice = $rate['total_charge'] / $convertRate;
```

Comment:
We should use division or multiplication here?

Reply:
We should use division. The responded value will be in the displayed(current) currency and we need to change it back to the base currency. The rate for base currency => current currency will be present. So we use the value divide the rate of base currency to current currency. (Magento will use the same rate to calculate the displayed value after this.)